extensions/amp-next-page/1.0/service.js

```diff
@@ -522,8 +517,8 @@ export class NextPageService {
    * @param {!Document} doc Document to attach.
    */
   sanitizeDoc(doc) {
-    // TODO(wassgha): Allow amp-analytics after bug bash
-    toArray(doc.querySelectorAll('amp-analytics')).forEach(removeElement);
+    // eslint-disable-next-line no-unused-vars
+    const analytics = toArray(doc.querySelectorAll('amp-analytics'));
```

Comment:
You should be able to rename the variable to `unusedAnalytics` to pass lint checks without having to disable it.